### DISCUSSION TITLE
Notes - 15th March 2024

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/reading-group-2024#22] [General]
## Lecture 7: Linear Equations and Matrix Algebra

- [ ] remove punctuation from equations
- [ ] @mmcky to look at `note` admonition (see what margin notes would look like)

## Lecture 6: Complex Numbers

- @pgrosser1 to add her notes here
- move whole lecture to be near Samuelson lecture
- review where `complex numbers` is used and cross link to new lecture location using a `seealso` admonition or to wikipedia

## Lecture 8: Eigenvalues and Eigenvectors

- @pgrosser1 to add her notes here
- ax.text() should include `$` signs for the math in figure labels
- add a note about representing vectors as a dot and sometimes as a dot with a line from the origin

## GDP Diagrams

- @pgrosser1 @mbek0605 @Jiarui-ZH to upload their latest edits
- @mmcky and @HumphreyYang to do a pass on the GDP project and tidy up the repo then pass on to @jstac 

**Next Week:** (Probability and Distributions Section) `Distributions and Probabilities` (@mbek0605, 10-15min) and `LLN & CLT (10-15min)` (@pgrosser1), `Monte Carlo and Option Pricing` (@Jiarui-ZH)

**Comment by pgrosser1** [comment]
My comments for Lecture 6 and 8 are included in the document attached below.
[QuantEcon Lecture 6 and 8 Notes.docx](https://github.com/QuantEcon/reading-group-2024/files/14694588/QuantEcon.Lecture.6.and.8.Notes.docx)

**Comment by HumphreyYang** [comment]
Hi @mmcky and @jstac,

Comments made by @pgrosser1 have been migrated to the corresponding issue tracker as well.
